css/css-flexbox/flexbox-min-width-auto-006.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<title>CSS Flexible Box Test: Aspect ratio handling of images</title>
+<link rel="author" title="Sergio Villar Senin" href="mailto:svillar@igalia.com" />
+<link rel="help" href="https://drafts.csswg.org/css-flexbox-1/#min-size-auto" />
+<link rel="match" href="reference/flexbox-min-width-auto-006-ref.html" />
+<meta name="assert" content="Test that min-width:auto does not incorrectly stretch items with aspect ratio" />
+
+<style>
+#reference-overlapped-red {
+    position: relative;
+    background-color: red;
+    width: 10px;
+    height: 10px;
+    top: 40px;
+    z-index: -1;
+}
+.outer {
```

Comment:
Better use names like in the previous test "constrained-width-flex" or something like that.